index.html

```diff
@@ -1210,25 +1218,25 @@ <h3>Types</h3>
         </table>
 
         <p>
-All <a>credentials</a>, <a>presentations</a>, and encapsulated objects MUST
-specify, or be associated with, additional more narrow <a>types</a> (like
-<code>UniversityDegreeCredential</code>, for example) so software systems can
-process this additional information.
+All <a>verifiable credentials</a>, <a>verifiable presentations</a>, and
```

Comment:
We meant credentials and presentations, specifically did not mean verifiable in this case.

Reply:
Fixed.